.github/workflows/build-docker-image.yml

```diff
@@ -0,0 +1,20 @@
+name: Build docker image
+
+on:
+  push:
+    branches: ['master']
+  pull_request:
+    branches: ['master']
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out repository
+      uses: actions/checkout@v3
+
+    - name: Build the Docker image
+      run: docker build . --file Dockerfile --tag itk-doxygen-docker:$(date +%Y-%m-%d-%H-%M)
```

Comment:
The resulting image name is e.g. `itk-doxygen-docker:2023-02-05-18-04` in https://github.com/InsightSoftwareConsortium/ITKDoxygen/actions/runs/4097813165/jobs/7066458749#step:3:1523

Not sure the colons are desirable, and whether the image should have an extension, e.g. `.img`. @blowekamp let me know your thoughts. Thanks.

Reply:
This is standard docker naming, imagename-colon-tag.

Reply:
It does not look like the image is being cached or push anyplace, so it doesn't matter too much.

Reply:
I know the artefact is not being stored anywhere. If that is something we should be doing, it may be done in a separate PR. I cared about the name for the sake of sanity.